### PR TITLE
enabling ast driver on ppc

### PR DESCRIPTION
**Pull request by RobbieAB**
I closed PR #5962 and created a new PR to more easily allow rewriting the commits involved to satisfy Zlogenes comments.

These changes are derived from things I needed to change to make it work on my Raptor Talos.

### CODE REVIEW

File: x11-drivers/xf86-video-ast/xf86-video-ast-1.1.5.ebuild [hunk 1]
```diff
@@ -6,6 +6,6 @@ EAPI=5
 inherit xorg-2
 
 DESCRIPTION="X.Org driver for ASpeedTech cards"
-KEYWORDS="amd64 x86 ~amd64-fbsd ~x86-fbsd"
+KEYWORDS="amd64 ppc ppc64 x86 ~amd64-fbsd ~x86-fbsd"
```
Comment (Zlogene) [inline x11-drivers/xf86-video-ast/xf86-video-ast-1.1.5.ebuild]: You should not add straight to stable keywords for a package you've just tested. So it should be 

`KEYWORDS="amd64 ~ppc ~ppc64 x86 ~amd64-fbsd ~x86-fbsd"`

Comment (RobbieAB) [reply x11-drivers/xf86-video-ast/xf86-video-ast-1.1.5.ebuild]: The reason for that, as noted in the commit message, was that the CI blocks the unmasking of the video_cards_ast use-flag if there is no copy of this driver available in stable. 

Going straight to stable seemed the least worst option, as this driver is already stable on amd64 and x86.

Comment (lu-zero) [reply x11-drivers/xf86-video-ast/xf86-video-ast-1.1.5.ebuild]: Surely it is better than making X ~ppc64, IMHO.

**Comment by mattst88** [comment]
I noticed you closed your original PR and opened a new one to respond to feedback. That isn't necessary -- you simply force push over your PR branch and github will automatically update the pull request.

Please use the imperative in commit messages. E.g., "Add ppc64 keywords" instead of "Adding ppc64 keywords"

Please move the use.mask entry to profiles/arch/powerpc/ppc64/use.mask to avoid adding ppc keywords.

I don't have a problem with adding stable ppc64 keywords. We have ~3 users. Waiting 30 days is not going to make a difference. But, if you wanted to add ~ppc64 keywords instead you could simply add video_cards_ast to profiles/arch/powerpc/ppc64/use.stable.mask.

**Comment by RobbieAB** [comment]
This PR should be put on hold or closed, as I have discovered that the AAST driver, while it appears to build on PPC, fails to load with an error about ioBase. Fixing this is likely to end up requiring an ebuild bump.

**Comment by RobbieAB** [comment]
I am going to close this pull request as fixing the AST driver is not going to happen by me soon.

**Pull request closed without merging**